javascript/javascript1/week2/studentmanager.js

```diff
@@ -0,0 +1,37 @@
+const class07students = [];
+function addStudentToClass(studentname){
+    if (studentname === "") {
```

Comment:
- Use [trim](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/Trim) to check if it is a long empty string like `"    "`